Documentation: Fix setup with loadbalancer expected output section

cryslam opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you -->

**What this PR does / why we need it**:
Fixes Installation with LoadBalancer documentation under the expected output section; wasn't reflecting results correctly after installation in terminal.

**Checklist:**
-   [x] Fixes Documentation
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

chandankumar4 commented on website/sidebars.json:
```diff
@@ -19,9 +19,12 @@
           "disk-loss",
-          "cpu-hog"
+          "cpu-hog",
+	  "drain-node"
```
Some indentation issue

ksatchit replied:
sidebars is showing up correctly in local website / view file also shows the correct indentation

chandankumar4 replied:
okay!!

chandankumar4 commented on website/sidebars.json:
```diff
@@ -19,9 +19,12 @@
           "container-kill",
           "pod-network-latency",
           "pod-network-loss",
+	  "pod-network-corruption.md",
```
is `.md` extension required?

ksatchit replied:
fixed!

PratikBorhade302 committed:
Fix #121: Broken link on what-is-litmus page (#122)

Fix #121.
Clicking on Sign you work should open: https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md#sign-your-work-with-developer-certificate-of-origin
Presently it opens the correct GitHub page but wrong Heading is mentioned in the link

Signed-off-by: Pratik Borhade <pratikborhade302@gmail.com>

imrajdas committed:
updating litmusctl installation link (#123)

Signed-off-by: rajdas98 <mail.rajdas@gmail.com>

PratikBorhade302 committed:
Fixes #124: Typo on chaos-workflow page (#125)

Signed-off-by: Pratik Borhade <pratikborhade302@gmail.com>

singh2005 committed:
removed duplicate words (#118)

Co-authored-by: Shubham Chaudhary <shubham@chaosnative.com>

serenashe committed:
Add a doc: Install Litmus on KubeSphere (#110)

* Add a doc: Install Litmus on KubeSphere

Signed-off-by: adakisme <serenashe@yunify.com>

* Update kubesphere-litmus.md

Co-authored-by: Shubham Chaudhary <shubham@chaosnative.com>

songrgg committed:
fix the mismatched description for chaos workflow (#130)

Signed-off-by: Songrong Jiang <songrong.jiang@booking.com>

Co-authored-by: Songrong Jiang <songrong.jiang@booking.com>

PratikBorhade302 committed:
Fixes #126: Missing links on chaos-workflow page (#127)

Signed-off-by: Pratik Borhade <pratikborhade302@gmail.com>

PratikBorhade302 committed:
Fix broken link and typo (#136)

* Fix #135: Broken link on run-your-first-workflow.md

Signed-off-by: Pratik Borhade <pratikborhade302@gmail.com>

* Typo on resources page

Signed-off-by: Pratik Borhade <pratikborhade302@gmail.com>

DarthBenro008 committed:
add: dex installation documentation for `litmusportal` (#134)

* feat: add dex installation documentation
* add: dex-integration concepts

Signed-off-by: DarthBenro008 <hkpdev008@gmail.com>

Jonsy13 committed:
Updated docs with v2.1.0 & v2.2.0 changes (#140)

* Added versioned docs for 2.1.0 & 2.2.0.

Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

Commit:
Added docs for upgrade agent (#141)

* Added docs for upgrade agent

Signed-off-by: Saranya-jena <saranya@chaosnative.com>

Commit:
updated flow charts by adding START and END steps (#128)

Signed-off-by: neelanjan00 <neelanjan@chaosnative.com>

Commit:
added chaos-operator and chaos-runner flow diagrams to chaos execution plane and centered monitoring diagrams (#129)

Signed-off-by: neelanjan00 <neelanjan@chaosnative.com>

Commit:
Functionality for Dark-Mode in Litmus Docs 2.0  (#131)

* enabled dark-mode for litmus-docs

Signed-off-by: neelanjan00 <neelanjan@chaosnative.com>

cryslam committed:
fix setup with loadbalancer docs
